eng/pipelines/aggregate-reports.yml

```diff
@@ -133,6 +132,25 @@ stages:
           displayName: 'Copy BOM report to artifact staging'
           condition: succeededOrFailed()
 
+        - task: AzureFileCopy@4
+          displayName: 'Upload maven site'
+          inputs:
+            SourcePath: '$(Build.ArtifactStagingDirectory)/staging/*'
+            azureSubscription: 'Azure SDK Artifacts'
+            Destination: AzureBlob
+            storage: azuresdkartifacts
+            ContainerName: 'azure-sdk-for-java'
+            blobPrefix: staging
+
+        - task: AzureFileCopy@2
+          displayName: 'Upload coverage report'
+          inputs:
+            SourcePath: '$(Build.ArtifactStagingDirectory)/staging/*'
```

Comment:
This looks to be uploading the exact same path as the first copy. Also why is this using the v2 of the task and the other using the v4 version?

Reply:
Thanks for pointing this out. I have it all wrong.